### PR TITLE
Fixed create schedule(issue #3, #4, #24)

### PR DESCRIPTION
**Pull request by shariq1501**
1) Moved  create schedule form in modal.
2) Added field validation on schedule create 
3) Displayed `Days  (Mon-Sun)` for days in  Schedules listing 
4) Added tooltip `ISO Days (1-7 Mon-Sun)` for `Days` label in create and update schedule
4) Added validation for blank name field (`only space`)  in update module

### CODE REVIEW

File: src/frontend/src/app/scheduler/create-schedule/create-schedule.component.html [hunk 1]
```diff
@@ -43,64 +44,59 @@
...
-        <div *ngIf="selected_schedule_type == 2" class="field" [ngClass]="{ 'has-error': f.submitted && invalidTime }">
+        <div [hidden]='!(selected_schedule_type == 2)' class="field">
           <label class="label">Time</label>
           <p class="control is-fullwidth">
-            <input id="time" class="input" type='text' name="time" [(ngModel)]="invalidTime" #time="ngModel" placeholder="hh:mm:ss" mask='00:00:00'
-              (ngModelChange)="invalidTime=false">
+            <input class="input" type='text' placeholder="hh:mm:ss" mask='00:00:00' formControlName="time">
           </p>
-          <div *ngIf="f.submitted && invalidTime" class="help is-danger level-left">Time must be in 00:00:00, 23:59:59 range</div>
+          <small *ngIf="form.controls.time.invalid && form.controls.time.touched" class="help is-danger level-left">&nbsp; Time must be in 00:00:00 or 23:59:59 range</small>
```
Comment (praveen-garg) [inline src/frontend/src/app/scheduler/create-schedule/create-schedule.component.html]: > in 00:00:00 or 23:59:59 range
`in range 00:00:00 - 23:59:59 `

File: src/frontend/src/app/scheduler/create-schedule/create-schedule.component.ts [hunk 2]
```diff
@@ -1,19 +1,16 @@
 import { Component, OnInit, EventEmitter, Input, Output } from '@angular/core';
 import { SchedulesService, AlertService } from '../../services/index';
+import { FormBuilder, FormGroup, Validators, FormControl } from '@angular/forms';
 import Utils from '../../utils';
+import { CustomValidator } from '../../directives/custom-validator';
```
Comment (praveen-garg) [inline src/frontend/src/app/scheduler/create-schedule/create-schedule.component.ts:5]: is this generic validator and can be used elsewhere too; or specific to CreateSchedule?

**Review by praveen-garg — COMMENTED** [review]
This PR needs a test with help of Amarendra Sinha

File: src/frontend/src/app/scheduler/create-schedule/create-schedule.component.ts (hunk 2 above)
Comment (shariq1501) [reply src/frontend/src/app/scheduler/create-schedule/create-schedule.component.ts:5]: This is generic and can be used anywhere per requirement.

**Comment by MonikaSharma06** [comment]
LGTM!

**Pull request merged**